x-pack/plugins/cases/server/saved_object_types/cases.ts

```diff
@@ -138,6 +138,12 @@ export const createCaseSavedObjectType = (
       },
       title: {
         type: 'text',
+        fields: {
+          keyword: {
+            type: 'keyword',
+            ignore_above: 160,
```

Comment:
We should remove it (my mistake, I know 🙂 ) in case it will cause a migration if we change it. The backend validation is enough. @rudolf if we change the `ignore_above` in the future will it trigger a migration (reindex)?

Reply:
Do you mean having the default value for this field?

Reply:
I think we would always reindex if any key was removed from the mappings (like removing the `ignore_above`) or if updating the mappings of the existing index fails.

Reply:
Thanks @rudolf! What I meant is if we changing the `ignore_above` value from `160` to `200` will trigger a migration? I am just curious 🙂. We will not add it (never had it before this PR) so we do not have this dilemma in the future.